src/liboslexec/runtimeoptimize.cpp

```diff
@@ -495,17 +495,17 @@ RuntimeOptimizer::turn_into_nop (int begin, int end, string_view why)
 
 void
 RuntimeOptimizer::insert_code (int opnum, ustring opname,
-                               const int *argsbegin, const int *argsend,
+                               const OIIO::cspan<int> args_to_add,
```

Comment:
Just FYI, you shouldn't need the namespace here, in oslconfig.h, it aliases span and cspan inside the OSL namespace to be OIIO::span/cspan.